src/components/layout.css

```diff
@@ -0,0 +1,622 @@
+html {
+  font-family: sans-serif;
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+body {
+  margin: 0;
+  -webkit-font-smoothing: antialiased;
```

Comment:
"webkit" is a ref to the tech that powers Chrome, Safari, and now Microsoft's latest browser. The idea here is that since different browsers implement CSS rendering differently, sometimes you have to target them specifically.